src/main/java/Duke/Task/TaskList.java

```diff
@@ -0,0 +1,71 @@
+package main.java.Duke.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    public ArrayList<Task> list;
+
+    public TaskList(ArrayList<Task> tasklist) {
+        this.list = tasklist;
+    }
+
+    /**
+     * Deletes the task from the Task List.
+     *
+     * @param taskNumber Task number to be deleted.
+     */
+    public String deleteTask(int taskNumber) {
+        Task taskDeleted = list.get(taskNumber);
+        list.remove(taskNumber);
+        return ("I have removed the task:\n" + taskDeleted.stringify() + "\n" + "Now you have "
+                + list.size() + " tasks in the list.");
+
+    }
+
+    /**
+     * Adds the task to the Task List
+     *
+     * @param task Task to be added.
+     */
+    public String addTask(Task task) {
+        list.add(task);
+        return
+                "I have added this task:\n"
+                        + task.stringify() + "\n"
+                        + "Now you have " + list.size() + " task(s) in the list.";
```

Comment:
The indentation here looks inaccurate. The first line of the String should follow the return statement above instead of being below it and indented again.